pyproject.toml

```diff
@@ -82,6 +84,12 @@ codespell_lib = [
 [tool.check-manifest]
 ignore = ["codespell_lib/_version.py"]
 
+# TODO: reintegrate codespell configuration after updating test cases
+#[tool.codespell]
+#builtin = ["clear","rare","informal","usage","code","names"]
+#ignore-words-list = ["uint"]
+#skip=[ "./.*","codespell_lib/data/*","codespell_lib/tests/*"]
```

Comment:
Is this for this PR, or a follow-up, or is this comment outdated and can be removed, or ...?

Reply:
I left it in pyproject.toml as a reminder tat IMHO the configuration should be ultimately there in that format.

Currently there are 2 issues:
1. The format is not recognized by codespell (see the other toml file);
2. The configuration impacts the test - which means that the tests are subject to the project codespell configuration itself.  So ideally the tests should not rely on the codespell configuration in the project's setup.cfg, pyproject.toml, ... .